docs/modules/operation/pages/deep-dive/admin/housekeeping/introduction.adoc

```diff
@@ -1,19 +1,19 @@
 
-[[housekeeping]]
 = Housekeeping Tasks
 
-There are a number of housekeeping tasks you may want to do regularly to ensure optimum system performance.
+There are a number of housekeeping tasks that you may want to do regularly to ensure optimum system performance.
 We also recommend you complete some of these housekeeping tasks before upgrading {page-component-title}.
 
 == Prior to upgrading
 
 [[prune-events]]
 === Prune unneeded events
+
 Use vacuumd or cron to regularly prune unneeded events.
 This helps to clean up your events table, and reduces the amount of data in your backups.
 We also recommend doing this before you begin the {page-component-title} upgrade process.
 
-For example, run the following to delete events older than six weeks that have no associated outages:
+For example, run the following code to delete events older than six weeks that have no associated outages:
```

Comment:
The following is part of the default vacuumd config, and is automatically run every 24hrs the server is up. It shouldn't need to be run manually.